resources/mutate-policy-with-flag-disabled.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: psp-user-group-disabled
 spec:
   policyServer: default
-  module: registry://ghcr.io/kubewarden/tests/psp-user-group:v0.1.1
+  module: registry://ghcr.io/kubewarden/tests/user-group-psp:v0.4.7
```

Comment:
does this mean that we can delete the old package `psp-user-group`?
https://github.com/orgs/kubewarden/packages/container/package/tests%2Fpsp-user-group

Reply:
I don't know if it's used for anything else than e2e tests - if not we can delete it